<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="createButton">
      <property name="text">
       <string>Create Document</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addVersionButton">
      <property name="text">
       <string>Add Version</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="listButton">
      <property name="text">
       <string>List Versions</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="diffButton">
      <property name="text">
       <string>Diff Versions</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
